README.md

```diff
@@ -44,7 +73,7 @@ support for the following annotations:
 * nginx.ingress.kubernetes.io/canary-weight-total
 
 If you are reliant on any annotations not listed above, you'll need to manually
-find a Gateway API equivalent. 
+find a Gateway API equivalent.
```

Comment:
still WIP, need to document how annotations should be treated

Reply:
I agree that we shouldn't translate annotations at all - we want to discourage annotation use on Gateway API objects as much as possible.